e2e-tests/insurance/cypress/e2e/journeys/application-submission/multiple-policy-type/buyer-conditions/submit-an-application-multiple-policy-type-buyer-minimal.spec.js

```diff
@@ -0,0 +1,32 @@
+import { APPLICATION } from '../../../../../../../constants';
+
+context('Insurance - submit an application - multiple policy type, minimal buyer', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.completeSignInAndSubmitAnApplication({
+      policyType: APPLICATION.POLICY_TYPE.MULTIPLE,
+      hasConnectionToBuyer: false,
```

Comment:
don't these all default to false?

Reply:
yes I think so, but it's clearer to have them all specifically marked as false, incase they change or other conditions are added. We do this in many application submission tests.